modulecheck-core/src/main/kotlin/modulecheck/core/UnusedPluginFinding.kt

```diff
@@ -22,27 +22,26 @@ import modulecheck.api.finding.Fixable
 import modulecheck.api.finding.Problem
 import modulecheck.api.finding.removeDependencyWithComment
 import modulecheck.api.finding.removeDependencyWithDelete
-import modulecheck.core.rule.KAPT_PLUGIN_FUN
-import modulecheck.core.rule.KAPT_PLUGIN_ID
 import modulecheck.parsing.gradle.Declaration
 import modulecheck.project.McProject
 import modulecheck.utils.LazyDeferred
 import modulecheck.utils.lazyDeferred
 import java.io.File
 
-data class UnusedKaptPluginFinding(
+data class UnusedPluginFinding(
```

Comment:
Made this generic. Can be used with any unused Gradle plugin

Reply:
This is good!

The kapt-oriented stuff is just about the oldest code in the project, and largely unchanged.  It doesn't quite fit the patterns of everything else, but it hasn't been bad enough for me to revisit it just yet.

At some point, I'm going to want to broaden the somewhat-dated KAPT checks to a more generic "code generation" scope which includes Anvil and KSP extensions.

So, this is a step in that direction. :)